app/services/workflows/non_passported_workflow.rb

```diff
@@ -53,17 +53,66 @@ def collate_and_assess_gross_income
       end
     end
 
+    class PersonDecorator
```

Comment:
`Decorator` in this codebase is used a lot, and up until now uniformly to name classes you can call `as_json` on. I know that this class is a decorator in the literal sense, but is there another name we can call it given that Decorator has this quite specific meaning?

Reply:
+1 - Actually this isn't a decorator either as it doesn't maintain the interface of the decorated object.